msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md

```diff
@@ -1,64 +1,111 @@
 ---
-title: Import External Platform Messages
-description: Learn how to use Microsoft Graph to import messages such as message history and data from an third-party platform to Teams.
+title: Import third-party platform messages to Teams using Microsoft Graph
+description: Learn how to use Microsoft Graph to import messages such as message history and data from any third-party platform to Teams.
 ms.localizationpriority: high
-author: "akjo"
+author: "surbhigupta"
 ms.topic: overview
-ms.owner: vishachadha
-ms.date: 09/02/2022
+ms.owner: mehakagarwal
+ms.date: 11/17/2025
 ---
 
 # Import third-party platform messages to Teams using Microsoft Graph
 
 With Microsoft Graph, you can migrate users' existing message history and data from an external system into a Teams channel. By enabling the recreation of a third-party platform messaging hierarchy inside Teams, users can continue their communications in a seamless manner and proceed without interruption.
 
 > [!NOTE]
-> In the future, Microsoft may require you or your customers to pay additional fees based on the amount of data imported.
+> In the future, Microsoft might require you or your customers to pay extra fees based on the amount of data imported.
 
-## Import overview
+## Permissions
 
-At a high level, the import process consists of the following:
+| ScopeName             | DisplayName                        | Description                                      | Type             | Admin Consent? | Entities/APIs covered |
+|-----------------------|------------------------------------|-------------------------------------------------|------------------|----------------|-------------------------|
+| Teamwork.Migrate.All  | Manage migration to Microsoft Teams| Creating and managing resources for migration to Teams. | **Application-only** | Yes            | POST /team             |
 
-1. [Create a team with a back-in-time timestamp](#step-1-create-a-team).
-1. [Create a channel with a back-in-time timestamp](#step-2-create-a-channel).
-1. [Import external back-in-time dated messages](#step-3-import-messages).
-1. [Complete the team and channel migration process](#step-4-complete-migration-mode).
-1. [Add team members](#step-five-add-team-members).
+> [!NOTE]
+> Delegated authentication isn't supported.
+>
+
+## Supported channel and chat types
+
+All new and existing channels and chats  support migration mode for importing historical messages. Migration mode is a special state that prevents certain operations during the data migration process to ensure data integrity.
```

Comment:
Add this line after note as a sub header and rules:
The migration mode:

* Temporarily restricts sending new messages to the channel or chat
* Prevents adding or removing members during migration
* Allows importing historical messages with custom timestamps
* Maintains the original conversation structure and hierarchy